src/main/java/uk/ac/ebi/embl/converter/cli/Main.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2025 EMBL - European Bioinformatics Institute
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package uk.ac.ebi.embl.converter.cli;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.LoggerContext;
+import io.vavr.Function0;
+import java.io.*;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+import picocli.CommandLine.*;
+import picocli.CommandLine.Model.CommandSpec;
+import uk.ac.ebi.embl.converter.Converter;
+import uk.ac.ebi.embl.converter.fftogff3.FFToGff3Converter;
+import uk.ac.ebi.embl.converter.ConversionError;
+import uk.ac.ebi.embl.converter.gff3toff.Gff3ToFFConverter;
+
+@Command(
+        name = "gff3tools",
+        subcommands = {CommandConversion.class, CommandLine.HelpCommand.class},
+        description = "Utility to convert and validate gff3 files")
+public class Main {
+    @Spec
+    CommandSpec spec;
+
+    private static final Logger LOG = LoggerFactory.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        int exitCode = -1;
+        try {
+            exitCode = new CommandLine(new Main()).execute(args);
+        } catch (Exception e) {
+            LOG.error(e.getMessage(), e);
+        }
+        exit(exitCode);
+    }
+
+    public static int exit(int code) {
+        System.exit(code);
+        return code;
+    }
+}
+
+// Using pandoc CLI interface conventions
+@Command(name = "conversion", description = "Performs format conversions to or from gff3")
+class CommandConversion implements Runnable {
```

Comment:
Do we need to do this as a subcommand?, I get that this is a good idea when there are going to be a lot of commands we need to perform